lib/rom/command.rb

```diff
@@ -142,6 +137,15 @@ def new(new_relation)
       self.class.build(new_relation, options.merge(source: relation))
     end
 
+    # Instance options
+    #
+    # @return [Hash]
+    #
+    # @api public
+    def options
+      @__options__
+    end
+
```

Comment:
yep, that's why this is WIP :P Just wanted to make it work. I think, I'll wrap initializer mixin with a module which includes instance methods, `#options` specifically